logging/config.go

```diff
@@ -175,14 +193,31 @@ func levelFromString(level string) (*zapcore.Level, error) {
 // when a config map is updated
 func UpdateLevelFromConfigMap(logger *zap.SugaredLogger, atomicLevel zap.AtomicLevel,
 	levelKey string) func(configMap *corev1.ConfigMap) {
+
 	return func(configMap *corev1.ConfigMap) {
-		loggingConfig, err := NewConfigFromConfigMap(configMap)
+		config, err := NewConfigFromConfigMap(configMap)
 		if err != nil {
 			logger.Errorw("Failed to parse the logging configmap. Previous config map will be used.", zap.Error(err))
 			return
 		}
 
-		level := loggingConfig.LoggingLevel[levelKey]
+		level, defined := config.LoggingLevel[levelKey]
+		if !defined {
+			// reset to global level
+
+			loggingCfg, err := zapConfigFromJSON(config.LoggingConfig)
+			switch {
+			case err == emptyLoggerConfigError:
+				level = zap.NewAtomicLevel().Level()
+			case err != nil:
+				logger.With(zap.Error(err)).Errorf("Failed to parse logger configuration. "+
```

Comment:
```suggestion
				logger.With(zap.Error(err)).Errorf("failed to parse logger configuration; "+
```

Reply:
Care to explain why? All other logged messages in this package start with a capital letter.

Reply:
Well, I guess you can fix the whole package.
From official golang page: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
It’s a log message, not an error string. `log.Errorf` is not `fmt.Errorf`.

Reply:
🤦‍♂ 
sorry about that :)

Reply:
No worries, thanks for the thorough review!